manifests/init.pp

```diff
@@ -37,6 +38,7 @@
   validate_string($service_name)
   validate_hash($vrrp_instance)
   validate_hash($vrrp_script)
+  validate_hash($vrrp_track_process)
```

Comment:
The next release will use Puppet 4 and so old `validate` functions are deprecated.

A datatype need to be used instead.
Details about this work in #163 .